working/v3.0.0-draft2/ciba-flow/CIBA-authorization-request-object-schema.json

```
@@ -0,0 +1,109 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://apicentre.paymentsnz.co.nz/schemas/CIBA-authorization-request-object-schema.json",
+  "type": "object",
+  "title": "The schema for the CIBA request object (OIDC/FAPI compliant)",
+  "additionalProperties": false,
+  "required": ["aud", "iss", "exp", "iat", "nbf", "jti", "scope", "ConsentId"],
+  "properties": {
+    "aud": {
+      "$id": "#/properties/aud",
+      "title": "The audiences for the token",
+      "description": "The value should be or include the client ID of the third party",
+      "oneOf": [
+        {
+          "type": "string",
+          "examples": ["https://api.alphanbank.com"],
+          "format": "uri"
+        },
+        {
+          "type": "array",
+          "$id": "#/properties/aud/items",
+          "items": {
+            "type": "string",
+            "pattern": "^(.+)$"
+          },
+          "examples": ["https://api.alphanbank.com"]
+        }
+      ]
+    },
+    "iss": {
+      "$id": "#/properties/iss",
+      "type": "string",
+      "title": "The Issuer of the token",
+      "description": "The value should be the client ID of the third party, unless signed by a different party",
+      "default": "",
+      "examples": ["s6BhdRkqt3"],
+      "pattern": "^(.+)$"
+    },
+    "exp": {
+      "$id": "/properties/exp",
+      "type": "integer",
+      "description": "Expiration time on or after which the ID Token MUST NOT be accepted for processing.",
```

```suggestion
      "description": "Expiration time on or after which the Request Object MUST NOT be accepted for processing.",
```

A couple more the same in this file.